docs/.vuepress/theme/global-components/CustomBadge.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+ <a :href="link" v-if="link"><Badge :type=thisType :text=text vertical="middle"/><OutboundLink  v-if="withLinkIcon"/></a>
+ <span v-else>
+  <Badge :type=thisType :text=text vertical="middle"/><OutboundLink  v-if="withLinkIcon"/>
```

Comment:
I think you might be missing quotes around some of your props: `:type="thisType" :text="text"`

Reply:
Well, thank you for your comment but I'm pretty sure that if I'd to this the result would be that the `text` prop that the `<Badge>` component receives will be literally "text", instead of passing down the `text` prop.